doc/terminal-v2-roadmap.md

```diff
@@ -26,12 +25,15 @@ Below is the schedule for when milestones will be included in release builds of
 | 2020-08-31 | [1.3] in Windows Terminal Preview<br>[1.2] in Windows Terminal | [Windows Terminal Preview 1.3 Release](https://devblogs.microsoft.com/commandline/windows-terminal-preview-1-3-release/) |
 | 2020-09-30 | [1.4] in Windows Terminal Preview<br>[1.3] in Windows Terminal | [Windows Terminal Preview 1.4 Release](https://devblogs.microsoft.com/commandline/windows-terminal-preview-1-4-release/) |
 | 2020-11-30 | [1.5] in Windows Terminal Preview<br>[1.4] in Windows Terminal | [Windows Terminal Preview 1.5 Release](https://devblogs.microsoft.com/commandline/windows-terminal-preview-1-5-release/) |
-| 2020-12-31 | [1.6] in Windows Terminal Preview<br>[1.5] in Windows Terminal | |
-| 2021-01-31 | 1.7 in Windows Terminal Preview<br>[1.6] in Windows Terminal | |
-| 2021-02-28 | 1.8 in Windows Terminal Preview<br>1.8 in Windows Terminal | |
-| 2021-03-31 | 1.9 in Windows Terminal Preview<br>1.9 in Windows Terminal | |
-| 2021-04-30 | 2.0 RC in Windows Terminal Preview<br>2.0 RC in Windows Terminal | |
-| 2021-05-31 | [2.0] in Windows Terminal Preview<br>[2.0] in Windows Terminal | |
+| 2021-01-31 | [1.6] in Windows Terminal Preview<br>[1.5] in Windows Terminal | [Windows Terminal Preview 1.6 Release](https://devblogs.microsoft.com/commandline/windows-terminal-preview-1-6-release/) |
+| 2021-03-31 | [1.7] in Windows Terminal Preview<br>[1.6] in Windows Terminal | [Windows Terminal Preview 1.7 Release](https://devblogs.microsoft.com/commandline/windows-terminal-preview-1-7-release/) |
+| 2021-04-30 | [1.8] in Windows Terminal Preview<br>[1.7] in Windows Terminal | |

```

Comment:
technically, 6 weeks later is

```suggestion
| 2021-04-15 | [1.8] in Windows Terminal Preview<br>[1.7] in Windows Terminal | |
```

or do we want to get that specific?

Reply:
I've just been using the milestone end date so we don't lock ourselves down too much

Reply:
I'd still prefer using the 15th on some of these tbh. It's really confusing reading some gaps as 1 month and others as 2.

Reply:
I'm cool with either, as long as they line up with when the milestone ends, which we usually put as the last day of the month.

Reply:
I mean, you can also update the milestone end dates too... If anything, I feel like the milestone end dates should align with this for consistency.

Reply:
It looks like this PR has been merged without this feedback being addressed. I came here from the doc, because I couldn't work out why there was a one month gap between some releases, and two months between others.